modules/jwt_auth_issuer/src/Controller/JwtAuthIssuerController.php

```diff
@@ -52,8 +59,10 @@ public function tokenResponse() {
       $response->error = "Error. Please set a key in the JWT admin page.";
       return new JsonResponse($response, 500);
     }
-
     $response->token = $token;
+    if ($this->moduleHandler()->moduleExists('jwt_auth_refresh')) {
```

Comment:
Would be nice to have a more elegant mechanism to hook into this then just doing a module check. Maybe an event listener or a plugin?